src/Symfony/Component/Messenger/Worker.php

```diff
@@ -13,6 +13,8 @@
 
 use Psr\EventDispatcher\EventDispatcherInterface;
 use Psr\Log\LoggerInterface;
+use Symfony\Component\Clock\Clock;
+use Symfony\Component\Clock\ClockInterface;
```

Comment:
We should depend on the PSR clock interface instead.

Reply:
not possible, this uses sleep

Reply:
note also that's it's very easy to turn a psr-clock into a symfony-clock, so that this doesn't create a too hard to match requirement: `new Clock($psrClock)` does it.